antora-ui-camel/src/css/header.css

```diff
@@ -351,6 +359,12 @@ body {
     margin-left: 2rem;
   }
 
+  @media screen and (min-width: 1024px) and (max-width: 1080px) {
+    .navbar-end {
+      margin-left: 1rem;
```

Comment:
This is to change the left margin between the top level navigation (i.e. "Blog") and the logo? I think we can make that for all resolutions.